2021q2/raidz-expansion.adoc

```diff
@@ -0,0 +1,30 @@
+= OpenZFS RAIDZ Expansion update
+
+Contact: Matthew Ahrens <matt@mahrens.org>
+
+Links: +
+link:https://github.com/openzfs/zfs/pull/12225[OpenZFS Pull Request] +
+link:https://youtu.be/3SUKJye54aI?t=6166[video from 2021 FreeBSD Developer Summit] +
+link:https://docs.google.com/presentation/d/1FeQgEwChrtNQBHfWSNsPK3Y53O5BnPh3Cz5nRa5GAQY/edit[slides from 2021 FreeBSD Developer Summit] +
+link:https://arstechnica.com/gadgets/2021/06/raidz-expansion-code-lands-in-openzfs-master/[news article from Ars Technica]
+
+=== Project
+
+This feature allows disks to be added one at a time to a RAID-Z group,
+expanding its capacity incrementally.  This feature is especially useful for
+small pools (typically with only one RAID-Z group), where there isn't
+sufficient hardware to add capacity by adding a whole new RAID-Z group
+(typically doubling the number of disks).
+
+FreeBSD's ZFS implementation comes from the OpenZFS project.  This work will integrate to the OpenZFS repo, with support for FreeBSD and Linux operating systems.
```

Comment:
"be integrated in" maybe?

Also, i'd s/support for/support for the/ (or maybe just make the whole phrase "support for FreeBSD and Linux" instead).

Reply:
sure thing.